packages/hop-node/src/cli/swap.ts

```diff
@@ -15,50 +15,72 @@ root
   .description('Swap tokens on Uniswap or via Hop AMM')
   .option('--chain <slug>', 'Chain', parseString)
   .option('--from <symbol>', 'From token', parseString)
+  .option('--from-native [boolean]', 'From native token', parseBool)
```

Comment:
>  a simple --from DAI --to hDAI seems to be pretty logical

not sure about that because it's unclear if `DAI` here is referring to `XDAI` or `WXDAI` since gnosis AMM only works with `WXDAI`

```bash
swap --chain gnosis --from DAI --to hDAI // unclear if swap is using XDAI or WXDAI
```

while this is more clear imo

```bsh
swap --chain gnosis --from-native --to hDAI // explicit about using XDAI for swap
```

I'm ok with removing `--from-native` and `--to-native` but change it use the exact tokens as cli arguments (`WXDAI`, `WMATIC`, `WETH`) for AMM swap on their native chains and `XDAI`, `MATIC`, `ETH` to refer to native `from` tokens which will wrap tokens before swapping

Reply:
Resolved by https://github.com/hop-protocol/hop/pull/398/commits/74d656ec6f3c838f2d866d0aa5e97ab78fcb5774

Reply:
Nice. I think this makes great sense 👍